locale/en/site.json

```diff
@@ -131,10 +131,10 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-09-15T16:00:00.000Z",
-      "endDate": "2022-09-30T23:00:00.000Z",
-      "text": "New security releases now available for 18.x, 16.x, and 14.x release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/september-2022-security-releases/"
+      "startDate": "2022-10-18T16:00:00.000Z",
+      "endDate": "2022-10-25T16:00:00.000Z",
+      "html": "<img alt='Node.js 19 is now live!' src='https://nodejs.org/static/images/NodeJS19_Banner.png'>",
+      "link": "https://nodejs.org/en/blog/release/v19.0.0/"
```

Comment:
I think in the past we've pointed to the announcement post rather than the changelog-style one.

I don't know the slug of that post yet - but it was https://nodejs.org/de/blog/announcements/v18-release-announce/ for 18.

Reply:
OK, I'll switch it to that format and if that doesn't end up being the right slug, we can update this again. 

```suggestion
      "link": "https://nodejs.org/en/blog/announcements/v19-release-announce/"
```